pkg/broker/logic.go

```diff
@@ -34,7 +34,7 @@ type BusinessLogic struct {
 	instances map[string]*exampleInstance
 }
 
-var _ broker.BusinessLogic = &BusinessLogic{}
+var _ broker.Interface = &BusinessLogic{}
```

Comment:
Not sure if we should rename `BusinessLogic` struct as well, didn't find the naming that bad, but can change?

Reply:
FWIW, I thought the struct name was fine. It's a placeholder name anyway in the starter pack, whereas the other value (broker.Interface) is a permanent name in the library.